assignment-client/src/entities/EntityTreeSendThread.cpp

```diff
@@ -280,7 +290,8 @@ void EntityTreeSendThread::startNewTraversal(const ViewFrustum& view, EntityTree
                     if (_entitiesInQueue.find(entity.get()) != _entitiesInQueue.end()) {
                         return;
                     }
-                    if (entity->getLastEdited() > startOfCompletedTraversal) {
+                    if (_knownState.find(entity.get()) == _knownState.end() ||
+                            (_knownState.find(entity.get()) != _knownState.end() && entity->getLastEdited() > _knownState[entity.get()])) {
```

Comment:
By inspection this check appears to be much more expensive than the old version (and this would still be true even if we optimized the multiple lookups into one), and it isn't clear to me that we need to bother with all of the new checks in this context.  I think there are some assumptions or misunderstandings (on my part or on @SamGondelman's) of what exactly is in the `_knownState` data.